mesonbuild/compilers/compilers.py

```diff
@@ -1291,6 +1291,11 @@ def build_unix_rpath_args(self, build_dir, from_dir, rpath_paths, build_rpath, i
                     paths = paths + ':' + padding
             args.append('-Wl,-rpath,' + paths)
 
+	# BBI_SOL11_4 BEGIN -- just to prevent code of get_compiler_is_linuxlike below -- linux is like sunos and not vice versa
+        if mesonlib.is_sunos():
+            return args
+	# BBI_SOL11_4 END
```

Comment:
__[Flake8]__ [E101] indentation contains mixed spaces and tabs ([link](https://sider.review/gh/repos/19784232/pulls/5350#issue-4029587))

<sub>posted by Sider</sub>